### PR TITLE
Automate driver release #4217

### PR DESCRIPTION
**Pull request by Aisha-ali-ibm**

### CODE REVIEW

File: developer_docs/release.md [hunk 1]
```diff
@@ -65,7 +66,9 @@ ls target/helm/repo
 
 The Docker image not been pushed by the previous build step so must be done manually, e.g.
 ```
-docker push ibmcom/restconf-driver:0.0.1
+ echo <IAMAPIKEY> | docker login --username iamapikey --password-stdin icr.io/ibm/cp4na-drivers/
+ docker push icr.io/ibm/cp4na-drivers
```
Comment (davetobin) [inline developer_docs/release.md:70]: Should be `icr.io/cp4na-drivers` in both above lines.

File: developer_docs/release.md [hunk 2]
```diff
@@ -65,7 +66,9 @@ ls target/helm/repo
...
+
 ```
 
 Complete the following:
```
Comment (davetobin) [inline developer_docs/release.md:74]: It looks like the release instructions are missing the automation steps and are just what would have been the fallback "Manual Approach".

File: pom.xml [hunk 3]
```diff
@@ -28,7 +28,7 @@
         <commons-io.version>2.7</commons-io.version>
         <docker-maven-plugin.version>1.2.2</docker-maven-plugin.version>
         <docker.directory>docker-ready</docker.directory>
-        <docker.registry>ibmcom</docker.registry>
+        <docker.registry>icr.io/ibm/cp4na-drivers</docker.registry>
```
Comment (davetobin) [inline pom.xml:31]: Should be `icr.io/cp4na-drivers`

**Review by davetobin — CHANGES_REQUESTED** [review]
Few changes required

File: developer_docs/release.md (hunk 1 above)
Comment (Aisha-ali-ibm) [reply developer_docs/release.md]: I've changed this to `icr.io/cp4na-drivers` now

File: developer_docs/release.md (hunk 2 above)
Comment (Aisha-ali-ibm) [reply developer_docs/release.md:103]: I've added in the missing steps.

File: pom.xml (hunk 3 above)
Comment (Aisha-ali-ibm) [reply pom.xml]: I've updated this also.

**Review by davetobin — APPROVED** [review]

**Pull request merged**